src/chainstate/stacks/db/blocks.rs

```diff
@@ -3870,13 +4019,34 @@ impl StacksChainState {
                 };
 
             let microblock_cost = clarity_tx.cost_so_far();
-            debug!("\n\nAppend block {}/{} off of {}/{}\nStacks block height: {}, Total Burns: {}\nMicroblock parent: {} (seq {}) (count {})\n", 
-                   chain_tip_consensus_hash, block.block_hash(), parent_consensus_hash, parent_block_hash,
-                   block.header.total_work.work, block.header.total_work.burn,
-                   last_microblock_hash, last_microblock_seq, microblocks.len());
+            debug!("\n\nAppend block";
+                   "block" => %format!("{}/{}", chain_tip_consensus_hash, block.block_hash()),
+                   "parent_block" => %format!("{}/{}", parent_consensus_hash, parent_block_hash),
+                   "stacks_height" => %block.header.total_work.work,
+                   "total_burns" => %block.header.total_work.burn,
+                   "microblock_parent" => %last_microblock_hash,
+                   "microblock_parent_seq" => %last_microblock_seq,
+                   "microblock_parent_count" => %microblocks.len());
+
+            // process stacking operations from bitcoin ops
+            let mut receipts = match StacksChainState::process_stacking_ops(
+                &mut clarity_tx,
+                stacking_burn_ops,
+                processed_total,
+            ) {
+                Err(e) => {
+                    let msg = format!("Invalid Stacks block {}: {:?}", block.block_hash(), &e);
```

Comment:
How can this error path be reached?  I'm not sure we want to punish a Stacks miner (i.e. invalidate the block) if someone sent a Stacking Bitcoin transaction that wouldn't have been a valid Stacks transaction, had it been sent to the Stacks chain.  The Stacks miner can't help what transactions get included in the Bitcoin chain.

Reply:
It unreachable code it turns out -- I'll patch `process_stacking_ops` to just return a receipt Vec

Reply:
Updated in 74da104fb